packages/runtime-common/realm.ts

```diff
@@ -1495,11 +1510,13 @@ export class Realm {
   private async readFileAsText(
     path: LocalPath,
     opts: { withFallbacks?: true } = {},
+    loader?: Loader,
```

Comment:
Maybe this is less relevant with types but it’s common for optional configuration to be the last argument so you don’t have to have `{}`

Reply:
This is now not relevant anymore with the latest changes - that param was removed.